src/container/container_iv.c

```diff
@@ -763,7 +761,7 @@ cont_iv_snapshots_fetch(void *ns, uuid_t cont_uuid, uint64_t **snapshots,
 	}
 
 	D_ALLOC(*snapshots,
-	      sizeof(iv_entry->iv_snap.snaps[0]) * iv_entry->iv_snap.snap_cnt);
+		sizeof(iv_entry->iv_snap.snaps[0]) * iv_entry->iv_snap.snap_cnt);
```

Comment:
(style)  line over 80 characters

Reply:
@jolivier23 I think this could probably use D_ALLOC_ARRAY(), do you agree?